test/action_controller/namespace_lookup_test.rb

```diff
@@ -19,6 +20,13 @@ class BookSerializer < ActiveModel::Serializer
             attributes :title, :body
 
             belongs_to :writer
+            has_many :chapters
+          end
+
+          class ChapterSerializer < ActiveModel::Serializer
+            attribute :title do
+              "Chapter - #{object.title}"
```

Comment:
When AMS doesn't find the serializer it uses `resource.as_json`. So I added this `"Chapter -"` part, to make sure that we use the namespaced serializer instead of `resource.as_json`.